content/home/examples/a-component-using-external-plugins.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Component Using External Plugins
+title: బాహ్య ప్లగిన్లు ఉపయోగించి ఒక భాగం
```

Comment:
I think the word ```component``` is a key term and should be just written as a component but not ```భాగం``` 

Totally changes the meaning

Reply:
```భాగం``` -> ```కంపోనెంట్```